package.json

```diff
@@ -66,6 +67,7 @@
     "composer.json": "composer validate --strict --no-check-all"
   },
   "dependencies": {
+    "@stripe/stripe-js": "^1.1.0",
```

Comment:
Using `^version` will trigger extra PRs from renovate bot which pins the deps version to `version`, so Iet's remove `^` in this PR changes. It's also not clear why we need this specific dependency, can you share some insights?

Reply:
Ah, sorry!

The TypeScript compiler will complain about missing functions if we don't include `@stripe/stripe-js`, which causes an error during type checking with `tsc --noEmit`. That's why I installed it.
I think the warning is because the Stripe React library has some declaration files (`.d.ts`) that include functions declared in `@stripe/stripe-js` and that's why we see the error.

I'm not sure if this can go in `devDependencies` or not though, but I installed it in `dependencies` just to be safe 👍 

> Using ^version will trigger extra PRs from renovate bot which pins the deps version to version, so Iet's remove ^ in this PR changes.

I didn't know that! Thanks for picking up on that; I'll make the change!

Reply:
Fixed in ec5a3f3ddab35b8cb7abc7318a5952c3f237d6f7